elements/pl-order-blocks/pl-order-blocks.js

```diff
@@ -59,7 +59,8 @@ window.PLOrderBlocks = function (uuid, options) {
   let sortables = optionsElementId + ', ' + dropzoneElementId;
   $(sortables).sortable({
     items: 'li:not(.info-fixed)',
-    cancel: '.info',
+    cancel: /* default */ 'input,textarea,button,select,option' +
+    /* PL-specific */ ',.pl-symbolic-input,.pl-string-input,.pl-integer-input,.pl-number-input',
```

Comment:
Is there an issue tied to this with more background context?

Reply:
I don't love this mostly because it means individual courses would have a hard time implementing custom elements that play nicely with `pl-order-blocks`. A potentially better option is to use a single class `.pl-order-blocks-allow-events` (or a data attribute `data-pl-order-blocks-allow-events`) that other elements would have to attach to their DOM elements to allow them to opt-in to this?

That said, backing up for a second...

> the problem is that the popover help text cannot be interacted with by default.

What exactly does this mean, that someone can't select/copy/paste help text, or something else?

Reply:
> the popover help text cannot be interacted with by default.

You can't see the help text at all, the button to see it doesn't work.

Reply:
>  it means individual courses would have a hard time implementing custom elements that play nicely with pl-order-blocks

Another alternative solution would be having a new option on the `pl-order-blocks` element that says which things to allow, something like `allow-nested=".pl-symbolic-input"` or whatever. This could potentially be nicer than having to put `data-pl-order-blocks-allow-events` in a whole bunch of places.

Reply:
> You can't see the help text at all, the button to see it doesn't work.

Even though `button` is in the `cancel` option? Is `cancel` not behaving how we expect it to?

> This could potentially be nicer than having to put data-pl-order-blocks-allow-events in a whole bunch of places.

Yeah, but now you have to remember to put `allow-nested=".pl-symbolic-input"` in a bunch of places, *and* we leak implementation details like the existence of a `.pl-symbolic-input` class 😛

Reply:
> Even though button is in the cancel option? Is cancel not behaving how we expect it to?
ya, I guess not...this is worth further investigation. 

> leak implementation details
oof, ya that's not good

Reply:
@nwalters512 Matt and I figured it out, the help text popovers are `a`s with `role="button"` and those aren't one the default list.

Reply:
Ahh! Nice. @mwest1066 is there any good reason for them to not just be standard `<button>`s?